rasa/shared/core/training_data/story_reader/yaml_story_reader.py

```diff
@@ -66,7 +66,7 @@ def from_reader(cls, reader: "YAMLStoryReader") -> "YAMLStoryReader":
             reader.template_variables,
             reader.use_e2e,
             reader.source_name,
-            reader.is_used_for_conversion,
+            reader.is_used_for_training,
```

Comment:
should be fine as `_` is just `protected`. Then you can also remove the `@property` thingy.
```suggestion
            reader._is_used_for_training,
```